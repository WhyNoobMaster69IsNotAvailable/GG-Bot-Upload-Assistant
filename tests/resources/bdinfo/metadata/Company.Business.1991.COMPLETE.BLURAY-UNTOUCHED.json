{
    "raw_video_file": "/data/downloads/FullDisk/Company.Business.1991.COMPLETE.BLURAY-UNTOUCHED/BDMV/STREAM/00002.m2ts",
    "raw_file_name": "Company.Business.1991.COMPLETE.BLURAY-UNTOUCHED",
    "largest_playlist": "00002.MPLS",
    "bdinfo_output_split": "b'Please wait while we scan the disc...\n# Group Playlist File Length Estimated Bytes Measured Bytes \n\n1 1 00002.MPLS 01:37:59 29,579,618,304 - \n2 2 00004.MPLS 00:21:07 4,215,441,408 - \n3 3 00001.MPLS 00:02:27 488,503,296 - \n4 4 00003.MPLS 00:01:55 525,723,648 - \n5 5 00000.MPLS 00:00:30 11,870,208 - \n'"
}
