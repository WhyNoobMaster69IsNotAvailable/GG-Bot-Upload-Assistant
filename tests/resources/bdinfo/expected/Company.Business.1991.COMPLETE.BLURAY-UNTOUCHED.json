{
    "video": [
        {
            "codec": "AVC",
            "bitrate": "34754 kbps",
            "resolution": "1080p",
            "fps": "23.976 fps",
            "aspect_ratio": "16:9",
            "profile": "High Profile 4.1"
        }
    ],
    "audio": [
        {
            "language": "German",
            "codec": "DTS-HD Master Audio",
            "channels": "2.0",
            "sample_rate": "48 kHz",
            "bitrate": "1564 kbps",
            "bit_depth": "16-bit"
        },
        {
            "language": "English",
            "codec": "DTS-HD Master Audio",
            "channels": "2.0",
            "sample_rate": "48 kHz",
            "bitrate": "1668 kbps",
            "bit_depth": "16-bit"
        }
    ],
    "label": "Company.Business.1991.COMPLETE.BLURAY-UNTOUCHED",
    "size": 32.43169876560569,
    "playlist": "00002.MPLS",
    "length": "1:37:59"
}
